lessons/migrations/0001_initial.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.0.7 on 2023-06-26 15:04
+
+from django.db import migrations, models
+import tinymce.models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Lesson',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=120)),
+                ('subset', models.CharField(max_length=120)),
```

Comment:
What is the difference between the "title" and "subset" fields?  Like as in how are they being used?